setup.py

```diff
@@ -35,6 +35,8 @@
            'scripts/yujin_tools_settings',
            'scripts/yujin_list_dirs_by_size',
            'scripts/yujin_list_git_branches',
+           'scripts/rosdep-generator/rosdep-generator',
```

Comment:
This one will be pointing to the wrong place...`scripts/rosdep-generator` I believe?